tests/e2e/commands_e2e/blockchain/deploy/suite.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package deploy
+
+import (
+	"fmt"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands_e2e/blockchain"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	deployTestJsonPath = "tests/e2e/commands_e2e/blockchain/deploy/deploy_tests.json"
+	subnetName         = "testSubnet"
+)
+
+var (
+	config *blockchain.TestJSONConfig
+	err    error
+)
+
+const ewoqEVMAddress = "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
+
+var _ = ginkgo.Describe("[Blockchain Deploy Flags]", ginkgo.Ordered, func() {
+	_ = ginkgo.BeforeEach(func() {
+		// Create test subnet config
+		commands.CreateSubnetEvmConfigSOV(subnetName, utils.SubnetEvmGenesisPath, ewoqEVMAddress)
```

Comment:
We use ewoq here because we use ewoq as validator manager owner when we deploy in local network